Content.Client/Lobby/LobbyState.cs

```diff
@@ -49,7 +52,16 @@ protected override void Startup()
 
             _voteManager.SetPopupContainer(Lobby.VoteContainer);
             LayoutContainer.SetAnchorPreset(Lobby, LayoutContainer.LayoutPreset.Wide);
-            Lobby.ServerName.Text = _baseClient.GameInfo?.ServerName; //The eye of refactor gazes upon you...
+
+            var lobbyNameCvar = _cfg.GetCVar(CCVars.ServerLobbyName);
+            Lobby.ServerName.Text = string.IsNullOrEmpty(lobbyNameCvar)
+                ? Loc.GetString("ui-lobby-title") + " "  + _baseClient.GameInfo?.ServerName
```

Comment:
Shouldn't the concatenation be done in the localization string instead of the order being hard-coded?

Reply:
I did not think a lot about that part or ever look at it again, because I don't imagine many servers will use the default. But yeah, fair. point